File: test/ayush.dev/python/pipelines/python_external_git_pipeline/code/python_external_git_pipeline/graph/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from python_external_git_pipeline.config.ConfigStore import *
from python_external_git_pipeline.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(
        col("c_tinyint"), 
        col("c_smallint"), 
        col("c_int"), 
        col("c_bigint"), 
        col("c_float"), 
        col("c_double"), 
        col("c_string"), 
        col("c_boolean"), 
        col("c_array"), 
        col("c_struct"), 
        concat(lit(Config.c_config1), col("c_smallint"), squared(col("c_int"))).alias("c1")
    )

File: test/randomtest/scallaexternalgitone/pipelines/SC_CONFIG_EXT_GIT_ONE/code/src/main/scala/io/prophecy/pipelines/sc_config_ext_git_one/graph/Reformat_1.scala
package io.prophecy.pipelines.sc_config_ext_git_one.graph

import io.prophecy.libs._
import io.prophecy.pipelines.sc_config_ext_git_one.udfs.PipelineInitCode._
import io.prophecy.pipelines.sc_config_ext_git_one.udfs.UDFs._
import io.prophecy.pipelines.sc_config_ext_git_one.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_1 {

  def apply(context: Context, in: DataFrame): DataFrame = {
    val Config = context.config
    in.select(
      concat(lit(Config.c_string),  col("`c   short  --`")).as("c_config_str"),
      concat(lit(Config.c_boolean), col("`c   short  --`")).as("c_config_bool"),
      concat(lit(Config.c_int),     col("`c   short  --`")).as("c_config_int"),
      concat(lit(Config.c_float),   col("`c   short  --`")).as("c_config_float"),
      concat(lit(Config.c_double),  col("`c   short  --`"))
        .as("c_config_double"),
      expr(Config.c_spark_expression).as("c_config_sparkexpression"),
      lookup("TestLookup", col("c_timestamp"))
        .getField("c-decimal")
        .as("c_lookup"),
      int_udf(col("`c   short  --`")).as("c_udf1")
    )
  }

}
